packages/ui/src/components/vuestic-components/va-modal/VaModal.demo.vue

```diff
@@ -283,22 +339,25 @@ export default {
       showModalSecondLay: false,
       showModalLongMessage: false,
       showModalWithoutTitle: false,
-      showModalWithoutTrasitions: false,
-      message: 'About 64% of all on-line teens say that do things online that they wouldn’t want their parents to know about.   11% of all adult internet users visit dating websites and spend their time Some of the classify their behavior as “cyber affair”   More then 60% of employees use company PC for the personal needs during their work hours as long as 80 minutes per day.',
+      showModalWithoutTransitions: false,
+      showModalNoDismiss: false,
+      showModalOverlay: false,
+      showModalOverlayOpacity: false,
+      showModalZIndex: false,
+      showModalCustomFooter: false,
+      message: this.$vb.lorem(),
+      longMessage: this.$vb.lorem(5000),
     }
   },
   methods: {
     onOk () {
-      // eslint-disable-next-line no-console
-      console.log('OK')
+      this.$vb.log('OK')
```

Comment:
That's mostly intended for in-template usage. In method you can just `console.log`.

Reply:
@asvae But it's cool - we don't need to disable `eslint`, for instance. No?

Reply:
Just saying what the intention was when I develop the thing.